xbmc/filesystem/FileDirectoryFactory.cpp

```diff
@@ -87,10 +87,10 @@ IFileDirectory* CFileDirectoryFactory::Create(const CURL& url, CFileItem* pItem,
         std::unique_ptr<CAudioDecoder> result = std::make_unique<CAudioDecoder>(addonInfo.second);
         if (!result->CreateDecoder() || !result->ContainsFiles(url))
         {
-          CLog::Log(LOGINFO,
-                    "CFileDirectoryFactory::{}: Addon '{}' support extension '{}' but creation "
-                    "failed (seems not supported), trying other addons and Kodi",
-                    __func__, addonInfo.second->ID(), strExtension);
+          CLog::LogF(LOGINFO,
+                     "Addon '{}' support extension '{}' but creation failed (seems not supported), "
+                     "trying other addons and Kodi",
+                     addonInfo.second->ID(), strExtension);
```

Comment:
At other places, for LOGINFO you removed the logging of the method signature, which I consider a good decision. INFO is normal operation not requiring this level of details. However, here you keep the log of method signature. I suggest to remove it, for consistency reasons.

Or is this log rather a WARNING or even an ERROR? In that case, the log type should be changed and method signature log should be kept.

Reply:
I agree that this should at least be considered a WARNING, as it shouldn't happen regularly. Changed.